src/windows/ui/main-window.ui

```diff
@@ -1655,6 +1655,18 @@
     <string>Export XML (Final Cut Pro)</string>
    </property>
   </action>
+  <action name="actionClearRecents">
+   <property name="icon">
+    <iconset theme="edit-clear" resource="../../images/openshot.qrc">
+     <normaloff>:/icons/Humanity/actions/16/edit-clear.svg</normaloff>:/icons/Humanity/actions/16/edit-clear.svg</iconset>
```

Comment:
Based on the icon used in Fedora (Adwaita theme), I'd say `edit-clear` is probably not a universally ideal choice.

![Screenshot from 2020-05-27 11-04-20](https://user-images.githubusercontent.com/538020/83037828-41aa0500-a00a-11ea-9478-e4db92188075.png)


Looking at the icon set, either `edit-clear-all` or `edit-delete` would be preferable:

![image](https://user-images.githubusercontent.com/538020/83037865-4a9ad680-a00a-11ea-9d34-4e0af85bc196.png)

Reply:
Hmm. Looks like Humanity doesn't have an `edit-clear-all`. So I guess `edit-delete` wins by default?

Reply:
(Humanity's `edit-delete` is this, which is "fine":)

![edit-delete](https://user-images.githubusercontent.com/538020/83039663-7028df80-a00c-11ea-8e46-c3aca761d075.png)